types/passport-auth-token/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
please have `strictNullChecks: true`, so that people who install this package with it turned on do not get unintended errors.

Reply:
I changed the value from false to true.